backend/services/deviceauth/devauth/devauth.go

```diff
@@ -30,12 +30,12 @@ import (
 	"github.com/mendersoftware/mender-server/pkg/plan"
 	"github.com/mendersoftware/mender-server/pkg/ratelimits"
 	"github.com/mendersoftware/mender-server/pkg/requestid"
+	tenant2 "github.com/mendersoftware/mender-server/pkg/tenant"
```

Comment:
There's no name conflict for `tenant` as far as I can see, can we remove the `2` suffix or use a more descriptive import name like `tenantClient`?
```suggestion
	"github.com/mendersoftware/mender-server/pkg/tenant"
```